src/sentry/dynamic_sampling/utils.py

```diff
@@ -10,14 +12,27 @@ class Bias(TypedDict):
 
 # These represent the biases that are applied to user by default as part of the adaptive dynamic sampling experience.
 # These can be overridden by the project details endpoint
+class RuleType(Enum):
+    UNIFORM_RULE = "uniformRule"
```

Comment:
i like this idea: small comment:
```
RuleType.UNIFORM
```
seems more organic, instead of `RuleType.UNIFORM_RULE`